tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py36
+envlist = py{27,35,36}-pandas{dev,21,20,19}
```

Comment:
This runs tox with the intended version https://travis-ci.org/tkf/dictsdiff/jobs/379117695#L440